archstrike-testing/teamsql/PKGBUILD

```diff
@@ -0,0 +1,25 @@
+# Maintainer: ArchStrike <team@archstrike.org>
+
+pkgname=teamsql
+pkgver=2.6.159
+pkgrel=2
+pkgdesc="Multi-Platform SQL Client"
+arch=('x86_64')
+license=()
+depends=("libappindicator" "libnotify" "gconf")
+optdepends=()
+options=()
+install=$pkgname.install
+source=("TeamSQL-${pkgver}.deb::https://dlpuop5av9e02.cloudfront.net/linux/stable/${pkgver}/TeamSQL-${pkgver}.deb"
+        'teamsql.sh')
+md5sums=('62b1b04053e05bdb0a984efb859d842d'
+         '7262cf67a1e2634401d6012d8d34e7d8')
+
+package() {
+  msg2 "Extracting the data.tar.xz..."
```

Comment:
msgs are unnecessary. remove all instances of msg2

Reply:
Removed